crowdsec-docs/unversioned/service_api/introduction.md

```diff
@@ -0,0 +1,39 @@
+---
+id: intro
+title: Introduction
+---
+
+:::info
+
+This feature is only accessible to paying users of Crowdsec.
```

Comment:
il faut dire "enterprise" users
Et c'est peut etre un peut TROP in your face ce info
Au moins le mettre sur une ligne et pas 3